classes/class-wc-connect-shipping-label.php

```diff
@@ -73,11 +73,32 @@ protected function get_packaging_data( WC_Order $order ) {
 			return $packages;
 		}
 
+		protected function get_selected_rates( WC_Order $order ) {
+			$shipping_method = reset( $order->get_shipping_methods() );
```

Comment:
The result of `$order->get_shipping_methods()` is what needs to be stored in a variable. `reset()` resets the internal array pointer, and expects to receive the variable by reference.